site2/docs/schema-evolution-compatibility.md

```diff
@@ -0,0 +1,165 @@
+
+* Example 2
+  
+    You want to load all Pulsar data into a Hive data warehouse and run SQL queries against the data. 
+
+    Same SQL queries must continue to work even the data is changed. To support it, you can evolve the schemas using the `BACKWARD` strategy.
+
+### FORWARD and FORWARD_TRANSITIVE 
+
+Suppose that you have a topic containing three schemas (V1, V2, and V3), V1 is the oldest and V3 is the latest:
+
+| Compatibility check strategy | Definition | Description |
+|---|---|---|
+`FORWARD` | Consumers using the **last schema** can process data written by producers using a new schema, even though they may not be able to use the full capabilities of the new schema. | The consumers using the schema V3 or V2 can process data written by producers using the schema V3. |
+`FORWARD_TRANSITIVE` | Consumers using **all previous schemas** can process data written by producers using a new schema. | The consumers using the schema V3, V2, or V1 can process data written by producers using the schema V3. 
+
+#### Example  
+  
+* Example 1
+  
+  Add a field.
+  
+  In most data formats, consumers written to process events without new fields can continue doing so even when they receive new events containing new fields.
+
+* Example 2
+  
+  If a consumer has an application logic tied to a full version of a schema, the application logic may not be updated instantly when the schema evolves.
+  
+  In this case, you need to project data with a new schema onto an old schema that the application understands. 
+  
+  Consequently, you can evolve the schemas using the `FORWARD` strategy to ensure that the old schema can process data encoded with the new schema.
+
+### FULL and FULL_TRANSITIVE 
+
+Suppose that you have a topic containing three schemas (V1, V2, and V3), V1 is the oldest and V3 is the latest:
+
+| Compatibility check strategy | Definition | Description | Note |
+|---|---|---|---|
+`FULL` |  Schemas are both backward and forward compatible, which means consumers using the last schema can process data written by producers using the new schema and consumers using the new schema can process data written by producers using the last schema. | The consumers using the schema V3 can process data written by producers using the schema V3 or V2, and the consumers using the schema V3 or V2 can process data written by producers using the schema V3. | For Avro and JSON, the default schema compatibility check strategy is `FULL`. <br>For all schema types except Avro and JSON, the default schema compatibility check strategy is `ALWAYS_INCOMPATIBLE`. |
+`FULL_TRANSITIVE` | The new schema is backward and forward compatible with all previously registered schemas. | The consumers using the schema V3 can process data written by the producers using the schema V3, V2 or V1, and the consumers using the schema V3, V2 or V1 can process data written by the producers using the schema V3. |
+
```

Comment:
@sijie I re-write this table, is it accurate?

Reply:
lgtm